### DISCUSSION TITLE
features/integrations

### DISCUSSION THREAD

**Discussion by giscus** [graphql-hive/console#1281] [Docs Discussions]
# features/integrations

GraphQL Hive Documentation

https://docs.graphql-hive.com/features/integrations

**Comment by mausch** [comment]
Where is this integration config?
Also what's the difference between this "github integration" and just using a token?
Are there things that can be done with the integration but not with a token?

**Reply by dotansimha** [reply]
The configuration for the integration is under the Hive app, under the Organization Settings. 
The token is used for many things, like pushing schemas, operations, and doing schema checks, while a GitHub integration is for the implementation of a CI pipeline that runs schema checks for PR. 
A regular token can do everything done in GitHub integration, but it will require you do manage the lifecycle manually. 
Also, with the integration to GitHub, you'll be able to connect a project to a repository and link commits to schema checks.
With the Slack integration, you'll be able to configure Slack alerts and notification when schema is pushed to Hive.

**Reply by mausch** [reply]
>  link commits to schema checks.

Ok, so this apparently is something that can be only done with the integration, and not with a token?

**Reply by dotansimha** [reply]
Yeah, exactly.
